tests/importer/importer.cpp

```diff
@@ -481,17 +481,17 @@ TEST(Importer, library)
     auto model4 = libcellml::Model::create("model4");
 
     EXPECT_TRUE(importer->addModel(model1, "a_model_the_first.cellml"));
-    EXPECT_TRUE(importer->addModel(model2, "b_tweede_model.cellml"));
-    EXPECT_TRUE(importer->addModel(model3, "c_troisieme_model.cellml"));
+    EXPECT_TRUE(importer->addModel(model2, "b_tweede_voorbeeld.cellml"));
+    EXPECT_TRUE(importer->addModel(model3, "c_troisieme_modele.cellml"));
```

Comment:
Accents, accents! Super important in French! :)

Reply:
Not in the ASCII range!  Super important in ... erm ... CellML ... :D

Reply:
?? It's a key, which can be an URL. So, accentuated characters ought to be fine...?